docs/user-guide/monitoring.md

```diff
@@ -0,0 +1,161 @@
+        regex: ([^:]+)(?::\d+)?;(\d+)
+        replacement: $1:$2
+
+      - source_labels: [__meta_kubernetes_service_name]
+        regex: prometheus-service
+        action: drop
+```
+
+```console
+$ kubectl create -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/monitoring/configuration.yaml
+configmap "prometheus-configuration" created
+```
+
+Now, the below yaml is used to deploy Prometheus in kubernetes:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prometheus-server
+  namespace: ingress-nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prometheus-server
+  template:
+    metadata:
+      labels:
+        app: prometheus-server
+    spec:
+      serviceAccountName: prometheus-server
+      containers:
+        - name: prometheus
+          image: prom/prometheus:v2.3.2
+          args:
+            - "--config.file=/etc/prometheus/prometheus.yml"
+            - "--storage.tsdb.path=/prometheus/"
+          ports:
+            - containerPort: 9090
+          volumeMounts:
+            - name: prometheus-config-volume
+              mountPath: /etc/prometheus/
+            - name: prometheus-storage-volume
+              mountPath: /prometheus/
+      volumes:
+        - name: prometheus-config-volume
+          configMap:
+            name: prometheus-configuration
+        - name: prometheus-storage-volume
+          emptyDir: {}
+```
+
+Now, run the following command to deploy prometheus in kubernetes:
+
+```console
+$ kubectl create -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/monitoring/prometheus.yaml
+clusterrole "prometheus-server" created
+serviceaccount "prometheus-server" created
+clusterrolebinding "prometheus-server" created
+deployment "prometheus-server" created
+service "prometheus-service" created
+```
+
+!!! Important: this example uses an `emptyDir` volume to store the prometheus database. This means once the pod dies you will lose all the data.
+
+### Prometheus Dashboard
+
+Now to open prometheus dashboard on Browser:
+
+```console
+kubectl get svc -n ingress-nginx
+NAME                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                                      AGE
+default-http-backend   ClusterIP   10.103.59.201   <none>        80/TCP                                       3d
+ingress-nginx          NodePort    10.97.44.72     <none>        80:30100/TCP,443:30154/TCP,10254:32049/TCP   5h
+prometheus-service     NodePort    10.98.233.86    <none>        9090:32630/TCP                               1m
+```
+
+Now obtain the IP address of the nodes in the cluster running:
+
+```console
+$ kubectl get nodes --selector=kubernetes.io/role!=master -o jsonpath={.items[*].status.addresses[?\(@.type==\"InternalIP\"\)].address}
+10.192.0.2 10.192.0.3 10.192.0.4
+```
+
+Now, open your browser and go to the following URL: _http://{node IP address}:{prometheus-svc-nodeport}_ to visit Prometheus Dashboard.
```

Comment:
All instructions steps start with "Now, ". It would sound better to me with if sentences simply started with a verb (eg. "Open this", "Obtain that"), but just a personal note.

Reply:
"*the* Prometheus dashboard"

Reply:
"visit the following URL"